src/internal/model_render.go

```
@@ -72,93 +73,124 @@ func (m *model) filePanelRender() string {
-		panelModeStringSmall = icon.Select
-		panelModeString = "Select"
+func (panel *filePanel) renderFileEntries(r *rendering.Renderer, mainPanelHeight, filePanelWidth int) {
+	if len(panel.element) == 0 {
+		r.AddLines(common.FilePanelNoneText)
+		return
 	}
 
-	// Only append Icon in case of nerdfont being true
-	if common.Config.Nerdfont {
-		panelModeString = panelModeStringSmall + icon.Space + panelModeString
+	end := min(panel.render + panelElementHeight(mainPanelHeight), len(panel.element))
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)